alongtheway.txt

```diff
@@ -518,7 +518,7 @@ Do you feel God speaking to you through Mary's story?
 
 Day 2: Mary's journey to Elizabeth's
 Read:  Luke 1:26-56, Matthew 1:18-25,
-Today we're reading about Mary's journey up to about the time Jesus was born. I want you to try to think of this from her point of view - not a Christmas-card picture-perfect romantic version - but not knowing how her story would be remembered. 
+Today we're reading about Mary's journey up to about the time Jesus was born. I want you to try to think of this from her point of view -- not knowing how her story would be remembered -- not a Christmas-card, picture-perfect, romantic version.
```

Comment:
I prefer it the original way or just take out the "Christmas card, picture perfect" bit altogether.

Sent from my iPhone

> On Jan 15, 2016, at 11:14 PM, Joshua Maxwell <notifications@github.com> wrote:
> 
> In alongtheway.txt:
> 
> > @@ -518,7 +518,7 @@ Do you feel God speaking to you through Mary's story?
> >  
> >  Day 2: Mary's journey to Elizabeth's
> >  Read:  Luke 1:26-56, Matthew 1:18-25,
> > -Today we're reading about Mary's journey up to about the time Jesus was born. I want you to try to think of this from her point of view - not a Christmas-card picture-perfect romantic version - but not knowing how her story would be remembered. 
> > +Today we're reading about Mary's journey up to about the time Jesus was born. I want you to try to think of this from her point of view -- not knowing how her story would be remembered -- not a Christmas-card, picture-perfect, romantic version.
> reorder a little bit?
> 
> —
> Reply to this email directly or view it on GitHub.
>